src/config/jwtConfig.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  JWT_SECRET: "jwt_secret",
```

Comment:
Change to JWT_SECRET: process.env.JWT_SECRET || "jwt_secret",

Reply:
Done